tests/python/vulture_test_files/unused_import.py

```diff
@@ -0,0 +1,2 @@
+from os import *  # Ignore PyUnusedCodeBear
```

Comment:
Exclude those test files in our coafile :+1: Not that we use VultureBear somewhen ;)